aesara/tensor/elemwise.py

```diff
@@ -1304,7 +1304,11 @@ def make_node(self, input):
             axis = list(range(inp_dims))
 
         copy_op = any(a < 0 for a in axis)
-        axis = np.core.numeric.normalize_axis_tuple(axis, ndim=inp_dims)
+        # normalize_axis_tuple always raises with ndim=0
+        try:
+            axis = np.core.numeric.normalize_axis_tuple(axis, ndim=max(1, inp_dims))
```

Comment:
Why not add a condition for `inp_dims == 0`?  This approach of using `ndims=1` when the number of dimensions is actually zero is a bit cryptic and holds the potential for real confusion if `normalize_axis_tuple` changes down the line.

Reply:
I did not want to duplicate the `normalize_axis_tuple` logic manually just for that case. Basically, these `Op`s treat scalar inputs as if they are 1D, so the `max(1, dims)` does not seem particularly bad. The ugly part is the `try/except` just to raise the correct `AxisError`. We can also add a test for `normalize_axis_tuple` to make sure the behavior hasn't changed, but I doubt they will make it do anything else for `ndim=0`.

What do you think?

Reply:
> Basically, these `Op`s treat scalar inputs as if they are 1D, so the `max(1, dims)` does not seem particularly bad.

Good point.